File: benchmark/results/ts-script-sonnet/run-001/b-baseline/output/src/domain/billing/services/TaxCalculator.ts
import { Invoice } from '../aggregates/Invoice';

/** TaxCalculator service. */
export class TaxCalculator {
  create(input: Partial<Invoice>): Invoice { throw new Error('not implemented'); }
  findById(id: string): Invoice | null { throw new Error('not implemented'); }
  list(limit: number): Invoice[] { throw new Error('not implemented'); }
  delete(id: string): void { throw new Error('not implemented'); }
}

File: benchmark/results/java-invocation-inline/run-005/b-baseline/output/com/metaengine/demo/billing/services/TaxCalculator.java
package com.metaengine.demo.billing.services;

import com.metaengine.demo.billing.aggregates.Invoice;
import java.util.List;

/** Tax computation service for the Invoice aggregate. */
public class TaxCalculator {

    /** Create a new Invoice with computed taxes from a partial input. */
    public Invoice create(Invoice input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find an Invoice by id; returns null if not found. */
    public Invoice findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List Invoices, capped at the supplied limit. */
    public List<Invoice> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete an Invoice by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}
